Sources/VLCMovieViewController.m

```diff
@@ -1383,7 +1435,11 @@ - (void)toggleUILock
     for (NSObject *item in items) {
         if ([item isKindOfClass:[UIControl class]]) {
             UIControl *control = (UIControl *)item;
-            control.enabled = !_interfaceIsLocked;
+            if (NEW_UI && item == _mediaNavigationBar.chromeCastButton) {
+                control.enabled = !_interfaceIsLocked && _vpc.renderer != nil;
```

Comment:
that looks wrong. The UILock should be not effected by having a renderer

Reply:
This is actually setting the chromecast button to be enabled only if there is a renderer device and the interface lock is not locked. aka if the interface is unlocked but no renderer it remains disabled